### PR TITLE
Fixes Build Issues on Mac

### LINKED ISSUE #169: WIP: Fix Build and Run issues on Mac
- [x] Fixed post build event errors.
- [x] Fix clang issues.
- [ ] OpenFileDialog

### PR DESCRIPTION
**Pull request by Seeker1437**
Adds a conditional post build event that uses `bash` in place of `call` for Mac users.

Fixes #169

### CODE REVIEW

**Comment by Seeker1437** [comment]
No longer WIP

File: AppData/PackageSources.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (danwalmsley) [inline AppData/PackageSources.json]: This file shouldn't be here, perhaps you have both your repo and the avalonstudio files in the same place?

**Comment by Seeker1437** [comment]
In gitter you asked:

>@Seeker1437 the mac updates
does it add a target?
the call doesn't work on linux

Can you elaborate on that question?

**Comment by Seeker1437** [comment]
Ready for review and testing.

**Pull request closed without merging**